<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_sex">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пол</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_str">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ag">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_iq">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_str">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сила</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dex">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ловкость</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_in">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Интеллект</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_luck">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_luck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удача</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_scr_lft">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_pts_lft">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Осталось очков</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clr">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_create">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>20</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_hp">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Здоровье</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hp_val">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Почти не осталость</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mp_val">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Что это?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mp">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Мана</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_def_val">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Можно попробовать убежать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_def">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Защита</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_atk_val">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Главное не поранить себя</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_atk">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Атака</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_class_val">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Задохлик</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_class">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Класс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_race">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Раса</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_race">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Люди</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Хоббиты</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Гномы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Орки</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_har">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Харизма</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_charm">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_priv">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Привлекательность</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_priv_val">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Я лучший?</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_sex_m">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Мужской</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_sex_w">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Женский</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>radioButton_sex_m</tabstop>
  <tabstop>radioButton_sex_w</tabstop>
  <tabstop>lineEdit_str</tabstop>
  <tabstop>lineEdit_ag</tabstop>
  <tabstop>lineEdit_iq</tabstop>
  <tabstop>lineEdit_charm</tabstop>
  <tabstop>lineEdit_luck</tabstop>
  <tabstop>comboBox_race</tabstop>
  <tabstop>lineEdit_scr_lft</tabstop>
  <tabstop>pushButton_clr</tabstop>
  <tabstop>pushButton_create</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
